postgres/postgres.go

```diff
@@ -0,0 +1,727 @@
+package postgres
+
+import (
+	"github.com/prismatik/jabba"
+)
+
+func Go() {
+	jabba.RunOrDie("echo", "'deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main'", ">", "/etc/apt/sources.list.d/pgdg.list")
```

Comment:
Nothing wrong with this, but would be nice to keep the way we add repos consistent between packages. In the docker package we write a file template whereas here we're echoing and redirecting the output. This shouldn't block merge, but we should decide which way we want to do it and then stick with that.

My vote is for file templates.